### ISSUE TITLE
PyTest failing with missing dependency

### ISSUE THREAD

**Issue by rgerkin** [pyrfume/pyrfume#38]
@schradert The package tests are failing due to a [missing dependency](https://github.com/pyrfume/pyrfume/runs/8125629481?check_suite_focus=true#step:5:16).  I'm not sure if `nbconvert` get removed in your PR or if that is an outstanding problem.  Could you check into this?

**Comment by rgerkin** [comment]
I need the package tests to pass before I draft a new release and upload to PyPI.

**Comment by schradert** [comment]
@rgerkin `nbconvert` was moved to the project's [`dev-dependencies`](https://github.com/pyrfume/pyrfume/blob/9a0d26489eaa6a44273507fb4a7db08ddb084c81/pyproject.toml#L38) in my PR, but I neglected to confirm this (and the other dependencies) would be installed correctly in the GitHub Action workflow, which it appears they aren't. I suspect this happens because the [dependency installation](https://github.com/pyrfume/pyrfume/blob/9a0d26489eaa6a44273507fb4a7db08ddb084c81/.github/workflows/pythonpackage.yml#L23) is not standard for `poetry` and doesn't resolve other dependency groups specified in `pyproject.toml`. I'm working on a PR right now to address this, following the examples [here](https://github.com/snok/install-poetry).
